unittest/unit/update-bcd.ts

```diff
@@ -861,5 +861,205 @@ describe('BCD updater', () => {
         version_added: null
       });
     });
+
+    describe('mirror', () => {
+      const browsers: any = {
+        chrome: {name: 'Chrome', releases: {85: {}, 86: {}}},
+        chrome_android: {
+          name: 'Chrome Android',
+          upstream: 'chrome',
+          releases: {86: {}}
+        }
+      };
+
+      const bcdFromSupport = (support) => ({
+        api: {FakeInterface: {__compat: {support}}}
+      });
+
+      const mirroringCase = ({support, downstreamResult}) => {
```

Comment:
This is a useful helper. Can you comment that it produces a report for Chrome Android 86? This is important for understanding the tests that use this, and his hidden in the UA string at the bottom.

Reply:
Certainly. Your comment also makes it clear that the UA string was a magic constant, so I've stored it in an appropriately-named variable to further underscore the intent of this code.